configs/ember-template-lint/package.json

```diff
@@ -32,6 +32,6 @@
     }
   },
   "engines": {
-    "node": "16.* || >= 18"
+    "node": "16.* || 18.* || >= 20"
```

Comment:
**TODO**

`16.*` will be removed once the packages that depend on this shared configuration drop Node 16 support.